app/services/exports/lettings_log_export_service.rb

```diff
@@ -8,13 +8,13 @@ def initialize(storage_service, logger = Rails.logger)
       @logger = logger
     end
 
-    def export_xml_lettings_logs(full_update: false)
+    def export_xml_lettings_logs(full_update: false, collection_year: nil)
       start_time = Time.zone.now
       daily_run_number = get_daily_run_number
       archives_for_manifest = {}
       base_number = LogsExport.where(empty_export: false).maximum(:base_number) || 1
       recent_export = LogsExport.order("started_at").last
-      available_collection_years.each do |collection|
+      available_collection_years(collection_year).each do |collection|
```

Comment:
not a major issue, I think this naming is slightly confusing - maybe a method called collection_years that returns [collection_year] if collection_year.present, and all_available_collection_years otherwise would be clearer?

Reply:
Renamed it, thanks!